smact/structure_prediction/utilities.py

```
@@ -1,5 +1,8 @@
 """Miscellaneous tools for data parsing."""
 
+
+
+from pymatgen.core.structure import Structure
```

Ruff

Move third-party import `pymatgen.core.structure.Structure` into a type-checking block

Move into type-checking block

(TCH002)